cmd/kusk/cmd/deploy.go

```diff
@@ -107,63 +108,68 @@ var deployCmd = &cobra.Command{
 				if err := k8sclient.Update(ctx, api, &client.UpdateOptions{}); err != nil {
 					return err
 				}
+				fmt.Printf("api.gateway.kusk.io/%s updated\n", api.Name)
 			} else {
 				return err
 			}
 		} else {
 			fmt.Printf("api.gateway.kusk.io/%s created\n", api.Name)
 		}
 
-		if watch {
-			var watcher *filewatcher.FileWatcher
-			absoluteApiSpecPath := file
-			if err != nil {
-				return err
-			}
+		if _, e := url.ParseRequestURI(file); e != nil {
+			if watch {
+				var watcher *filewatcher.FileWatcher
+				absoluteApiSpecPath := file
```

Comment:
Strictly speaking do we need to declare another variable - `absoluteApiSpecPath` - can we just not use `file` directly?